style.css

```diff
@@ -685,3 +787,304 @@ ul {
     gap: 30px;
   }
 }
+
+.close-btn {
+  align-self: flex-end;
+  background-color: transparent;
+  border: none;
+  margin: 10px;
+}
+
+.popup-window {
+  display: none;
+  flex-direction: column;
+  height: 100vh;
+  width: 100vw;
+  position: fixed;
+  top: 0;
+  left: 0;
+  overflow-y: scroll;
+  background-color: #fff;
+  z-index: 10;
+}
```

Comment:
- [ ] Kindly make sure to align correctly your popUp contents in mobile version and make sure to remove overflow which appearing in order to meet the provided Figma design. :wink: 

   - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/202391042-6a01345d-33e0-4575-b74a-34c26ec8a349.png)

   - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/202391152-921b4faa-556d-48ea-9de2-f23175c7fce7.png)